docs/simulator_binaries.md

```diff
@@ -1,14 +0,0 @@
-# How to download simulators binaries from S3 via command line
```

Comment:
I'd keep somewhere the information on the simulation binaries. You can refer to deployment documentation for more details.

Reply:
At some points the developer should know how to replace downloaded binary with their own copies and that should be documented

Reply:
I moved the information from this page, to the first point of `Building and running the app` in  `For developers`. You suggested it in your first comment. Should I also keep this file?

Reply:
ITs ok now